extensions/amp-story-audio-sticker/0.1/amp-story-audio-sticker.js

```diff
@@ -101,11 +107,54 @@ export class AmpStoryAudioSticker extends AMP.BaseElement {
     return Services.storyStoreServiceForOrNull(this.win).then(
       (storeService) => {
         this.storeService_ = storeService;
-        this.initializeListeners_();
+        this.element.addEventListener('click', this.onClick_.bind(this), true);
+        this.storeService_.subscribe(
+          StateProperty.MUTED_STATE,
+          this.onMutedStateChange_.bind(this)
+        );
       }
     );
   }
 
+  /**
+   * When the sticker is clicked, toggle muted state and highlight
+   * the mute control on system layer.
+   * @private
+   */
+  onClick_() {
+    this.storeService_.dispatch(Action.TOGGLE_MUTED, false);
+    this.getSystemLayerEl_().classList.add(
+      'i-amphtml-story-highlight-mute-audio-control'
+    );
+  }
+
+  /**
+   * Disable highlight once the story gets muted again.
+   * @private
+   * @param {boolean} isMuted
+   */
+  onMutedStateChange_(isMuted) {
+    if (isMuted) {
+      this.getSystemLayerEl_().classList.remove(
+        'i-amphtml-story-highlight-mute-audio-control'
+      );
+    }
+  }
+
+  /**
+   * Get the system layer element.
+   * @private
+   * @return {HTMLElement}
+   */
+  getSystemLayerEl_() {
+    return (
+      this.systemLayerEl_ ??
+      this.element
+        .closest('amp-story')
+        .querySelector('.i-amphtml-system-layer-host')
+    );
```

Comment:
If you want to cache it, you need to do this:
```suggestion
      (this.systemLayerEl_ = this.element
        .closest('amp-story')
        .querySelector('.i-amphtml-system-layer-host')
    ));
```